GDJS/Runtime/runtimeobject.ts

```diff
@@ -1253,6 +1256,37 @@ namespace gdjs {
       return this.hitBoxes;
     }
 
+    /**
+     * Get the hit boxes for the object around a given area.
+     * The hit boxes don't need to actually overlapping the area,
+     * but the ones that do must be returned.
+     *
+     * The default implementation returns the same as {@link getHitBoxes}.
+     *
+     * This method can be override by grid based objects to optimize collision checks.
+     *
+     * When overriding this method, the following ones should be overridden too:
+     * * {@link getHitBoxes}
+     * * {@link getAABB}
+     * * {@link updateHitBoxes}
+     * * {@link updateAABB}
+     *
+     * @param left bound of the area
```

Comment:
Let's precise "left bound of the area, in scene coordinates" and for the 3 others too :)